app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -519,8 +519,7 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         clientBuilder = new ClientBuilder();
-        prefs = getSharedPreferences(Constant.THEME, MODE_PRIVATE);
-        if(prefs.getString(Constant.THEME,"Light").equals("Dark")) {
+        if(PrefManager.getTheme().equals("Dark")) {
```

Comment:
Comparing theme to "Dark" where as in contant class it is defined as "dark". Use Constant.DARK here instead of hardcoding the string. Define one set, either "Light" and "Dark" or "light" and "dark".